q2_types/ordination/_transformer.py

```diff
@@ -7,11 +7,26 @@
 # ----------------------------------------------------------------------------
 
 import skbio
+import pandas as pd
+import qiime2
 
 from ..plugin_setup import plugin
 from . import OrdinationFormat
 
 
+def _ordination_format_to_ordination_results(ff):
+    return skbio.OrdinationResults.read(str(ff), format='ordination',
+                                        verify=False)
+
+
+def _ordination_format_to_dataframe(ff):
+    ordination_results = _ordination_format_to_ordination_results(ff)
+    df = ordination_results.samples
+    df.index.name = 'ID'
```

Comment:
Can you make the index name "Sample ID", in order to distinguish between sample and feature ordination coordinates? Both can be stored in the `OrdinationResults` object under different dataframe attributes.